### PR TITLE
Add pikchr@1.0.0

### PR DESCRIPTION
**Pull request by jmillikin**

### CODE REVIEW

File: modules/pikchr/1.0.0/source.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://mirror.bazel.build/github.com/drhsqlite/pikchr/archive/refs/tags/version-1.0.0.tar.gz",
```
[the remark this answers is not in the document]
Comment (jmillikin) [reply modules/pikchr/1.0.0/source.json:2]: Is there any particular reason why the module should use an unstable upstream source location?

I've had bad experiences in the past with Bazel builds depending on GitHub's auto-generated tarballs, for example the breakages documented in https://github.com/protocolbuffers/protobuf/issues/3894

If it's BCR policy to use unstable source URLs for GitHub projects without releases, then I can do that, but it's different from what I would expect you to want.

[the remark this answers is not in the document]
Comment (meteorcloudy) [reply modules/pikchr/1.0.0/source.json:2]: The main reason is that `https://mirror.bazel.build` is not intended to be used as a mirror for BCR modules (due to potential legal issues), it's mostly used by Bazel itself. You can host your own mirror of the source archive somewhere and add it to the `bazel_registry.json` config or just use `--experimental_downloader_config` . The source repo check should have caught this earlier, but it seems there is a bug.

[the remark this answers is not in the document]
Comment (jmillikin) [reply modules/pikchr/1.0.0/source.json:2]: The source repo check was changed by https://github.com/bazelbuild/bazel-central-registry/pull/2754 to allow `mirror.bazel.build` URLs for GitHub source archives.

Should that PR be reverted?

[the remark this answers is not in the document]
Comment (meteorcloudy) [reply modules/pikchr/1.0.0/source.json:2]: Oh I see, please do! /cc @fmeum

[the remark this answers is not in the document]
Comment (meteorcloudy) [reply modules/pikchr/1.0.0/source.json:2]: That means you have to also fix all modules that are currently using mirror.bazel.build URLs.

[the remark this answers is not in the document]
Comment (fmeum) [reply modules/pikchr/1.0.0/source.json:2]: Sorry, I didn't know we aren't supposed to use the mirror like that.

[the remark this answers is not in the document]
Comment (meteorcloudy) [reply modules/pikchr/1.0.0/source.json:2]: Np! We tried to make it work for BCR but eventually got shot down by the lawyer. Probably should have documented the decision somewhere.

**Pull request closed without merging**